CONTRIBUTING.md

```diff
@@ -131,12 +130,33 @@ The steps for development are:
 4. Run `renv::snapshot()` at the end of your session to capture the additional dependencies. *Be careful if `renv::snapshot()` suggests removing packages!* If there have been additions to `renv.lock` in another branch while you were working, you may need to run `renv::restore()` again before `renv::snapshot()`.
 5. If there are dependencies you might want that are not captured automatically by `renv::snapshot()` (this may happen if a package is "recommended" by another, but not required), add them to `components/dependencies.R` with a call to `library()` and an explanatory comment. Then rerun `renv::snapshot()`
 6. Commit any changes to `renv.lock` and `dependencies.R`.
-7. File a pull request with _only_ the changes to the `renv.lock` and `dependencies.R`  before other changes. 
+7. File a pull request with _only_ the changes to the `renv.lock` and `dependencies.R`  before other changes (see next section for tips on creating this PR).
 This is necessary because the automated render testing we do will fail if the Docker image has not been updated (see '[Pushing to Dockerhub via GitHub Actions](#pushing-to-dockerhub-via-github-actions)' below).
 
 Note that when you open up the `training-modules.Rproj`, the `.Rprofile` file makes it such that the `renv` library is loaded and the repositories in the `renv.lock` file will be set with `options(repos = ...)`.
 
+### Steps for creating renv.lock only changes pull requests
+
+For most cases you can create your `renv.lock-only` changes PR by following these steps:
+
+*Step 1)* Create your `renv.lock-only` branch from the latest `master` branch.  
```

Comment:
Nit-picky I realize, but can we match the style in the rest of the document here? (e.g., make this `1.` with no italics)